libraries/Wire/src/utility/twi.c

```diff
@@ -396,6 +424,40 @@ void twi_releaseBus(void)
   twi_state = TWI_READY;
 }
 
+/* 
+ * Function twi_setTimeoutInMicros
+ * Desc     set a global timeout for while loops that we might get stuck in
+ * Input    timeout value in microseconds
+ * Output   none
+ */
+void twi_setTimeoutInMicros(uint16_t timeout)
```

Comment:
Should be `void twi_setTimeoutInMicros(unsigned long)`

Reply:
also: uint32_t